assignment/core/src/main/java/com/assignment/core/workflows/PageCreationWorkflowProcess.java

```diff
@@ -0,0 +1,78 @@
+package com.assignment.core.workflows;
```

Comment:
How will the workflow run on AEM instance? We cannot manually create models on AEM instance. Add the workflow design model and runtime model in the code.